src/main/java/com/wynntils/antiope/core/DiscordGameSDKCore.java

```diff
@@ -55,6 +55,25 @@ public static void loadLibrary() {
         if (osName.contains("windows")) {
             osName = "windows";
             objectName = LIBRARY_NAME + ".dll";
+
+            // discord game sdk needs to be loaded first on windows
+            if (arch.equals("amd64")) arch = "x86_64"; // we don't want to rename the game sdk's internals, so we need to rename our arch temporarily
+            InputStream discordInputStream = DiscordGameSDKCore.class.getResourceAsStream("/discord_game_sdk/lib/" + arch + "/discord_game_sdk.dll");
+            if (discordInputStream == null) {
+                throw new RuntimeException("Could not find discord_game_sdk.dll in classpath");
+            }
+            File discordTempDirectory = FileUtils.createTemporaryDirectory();
+            File discordTempFile = new File(discordTempDirectory, "discord_game_sdk.dll");
+            discordTempDirectory.deleteOnExit();
+            discordTempFile.deleteOnExit();
+
+            try {
+                Files.copy(discordInputStream, discordTempFile.toPath());
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+
+            System.load(discordTempFile.getAbsolutePath());
```

Comment:
Oh so we load the original SDK first on windows, then our modified one... Okay. I think there could be a method made for creating a temporary copy of a file from the classpath to make this code more readable but I can do that as a future improvement.